### DISCUSSION TITLE
Following quickstart on OS X, can't reach hubs service in the browser

### DISCUSSION THREAD

**Discussion by truedat101** [mozilla/hubs#2940] [General Questions & Discussion]
**Description**
Cannot get the service to load from a browser

**To Reproduce**
Perform these steps:
https://github.com/mozilla/hubs#quick-start
open a browser to : https://localhost:8080
no page is loaded

**Expected behavior**
Should open a hubs service

**Screenshots**
N/A blank page loads

**Hardware** 
 - Device: desktop
 - OS: 10.12
 - Browser Firefox

**Additional context**
I've tried these same steps on OS X and windows.  I get the same result.

Are there some additional steps needed for local development?  Running witn node.js 12, and follow steps in quickstart.

**Comment by truedat101** [comment]
triaging: similar post on SO: https://stackoverflow.com/questions/62337760/how-to-run-mozilla-hubs-on-my-local-machine

I will try to run through the complete setup with https://github.com/mozilla/reticulum to see if I've missed a step at all.

**Comment by truedat101** [comment]
On linux, had no problem getting this up and going.  Something wrong with the setting of hubs.local in the hostfile?  Maybe some special instructions for that on OS X?  Or something else binding on the port.

**Reply by pattersonbl2** [reply]
Are you still have issues with this.  I have deploy on OSx without any issue or special steps
